pkg/cmd/server/crypto/crypto.go

```diff
@@ -147,8 +155,11 @@ func GetCA(certFile, keyFile, serialFile string) (*CA, error) {
 	}, nil
 }
 
-func MakeCA(certFile, keyFile, serialFile, name string) (*CA, error) {
+func MakeCA(certFile, keyFile, serialFile, name string, out io.Writer) (*CA, error) {
```

Comment:
1. Is this going to be used for anything but creating the openshift CA?
2. Where else could we reasonably do it? This can be called via EnsureCA which is a similar utility method that decides whether or not to write a new CA. Higher-level logic doesn't necessarily know whether the CA will be written. It seems to make sense to do it here if you ever want users to know the CA has been (over)written.
3. Output seems unlikely to cause problems, but if you were using this and really didn't want it, just pass a nil writer.